### DISCUSSION TITLE
Backend  Asp.net 6 Clean Architecture implementation with CQRS

### DISCUSSION THREAD

**Discussion by syfch-dev** [realworld-apps/realworld#824] [WIP Implementations]
Hello everyone I'm gonna create .net core 6 clean architecture implementation Thanks

**Comment by aji-emmanuel** [comment]
Hey, @seyfullahcuhadar  Just stumbled on this idea and I am also interested in making an asp.net implementation. I'm not so sure how this works. Hoping we could chat about it. Will contact you on linkedIn.
